src/main/java/bio/terra/service/profile/google/GoogleBillingService.java

```diff
@@ -81,8 +81,8 @@ public boolean canAccess(String billingAccountId) {
             .setResource(resource.toString())
             .addAllPermissions(permissions)
             .build();
-        try {
-            TestIamPermissionsResponse response = cloudBillingClient().testIamPermissions(permissionsRequest);
+        try (CloudBillingClient client = cloudBillingClient()) {
```

Comment:
Not related to the bug, but thank you for this.

Reply:
but of course!